src/App/App.js

```diff
@@ -0,0 +1,17 @@
+import { Routes, Route } from 'react-router-dom';
+import AllHorses from '../AllHorses/AllHorses';
+import Dashboard from '../Dashboard/Dashboard';
+import HorseProfile from '../HorseProfile/HorseProfile';
+import './App.css';
+
+function App() {
+  return (
+    <Routes>
+      <Route path='/' element={<Dashboard />} />
+      <Route path='/horses' element={<AllHorses />} />
+      <Route path='/horses/:id' element={<HorseProfile />} />
```

Comment:
Looks good! Definitely how I was planning to do the routes. 
We may want to add a 404 page, which we can do with the "*" thing I believe. And then just make a simple page not found component to route to.